lab-cameron/README.md

```diff
@@ -0,0 +1,6 @@
+## Vanilla REST api
```

Comment:
Good, if brief README, but good job for putting specific instructions telling a user how to use your API. 👍